system/core-workflows.rst

```diff
@@ -1,16 +1,16 @@
 Core Workflows
 ==============
 
-Core Workflows allow you to create complex ticket processes just as
-(but not limited to):
+Core Workflows allow you to adjust objects in many ways.
```

Comment:
`objects`? Or, perhaps, `object attributes`?

In the end, Core Workflow applies to forms (masks) only, that normally contain fields which allow manipulation of object attributes.